### PR TITLE
Test no coverage mask

### PR DESCRIPTION
**Pull request by nick-pestell**
This PR implements a bug fix from @ellisrichardj which avoids zero coverage regions being called REF in consensus file output. 

### Filtering zero coverage

This is enabled by an extra command in `mask.bash` (line 37) which calls `bedtools genomecov` and searches the `.bam` file for regions of 0 coverage. The output `zerocov.bed` is merged with `quality_mask.bed` and the repeat mask file to create a set of positions to be filtered from the consensus.

### Testing new filter

This new filtering method is tested in `test_mask.py`. A test reference genome (`tests/data/tinyref.fas`) is of length 20 bases. By parsing in a test `.sam` file (`tests/data/edge-cases.sam`) which contains a single read of length 19 (one fewer than the reference), and a VCF file with 19 entries to `mask.bash`, and comparing the output `.bed` file to the ideal outcome.

### CODE REVIEW

File: tests/unit_tests/mask_tests.py [hunk 1]
```diff
@@ -17,17 +17,22 @@ def test_mask(self):
...
         masked_filepath = self.temp_dirname+'masked.bed'
         regions_filepath = self.temp_dirname+'regions.bed'
+        sam_filepath = self.temp_dirname+'edge-cases.sam'
+        bam_filepath = self.temp_dirname+'edge-cases.bam'
         shutil.copy('./tests/data/edge-cases.vcf', vcf_filepath)        
         shutil.copy('./tests/data/tinymask.bed', rpt_mask_filepath)        
+        shutil.copy('./tests/data/edge-cases.sam', sam_filepath)        
         
         self.gzip(vcf_filepath, vcf_gz_filepath)
+        self.sam_to_bam(sam_filepath, bam_filepath)
```
Comment (ellisrichardj) [inline tests/unit_tests/mask_tests.py:27]: Is it worth storing the `edge-cases.bam` file in the repo to save having to do the conversion each time?

**Review by ellisrichardj — APPROVED** [review]
All looking good Nick - thanks for getting the test in place.  Just one small comment (which you can choose to ignore!)
